03-task-manager/starter/contollers/tasks.js

```diff
@@ -0,0 +1,52 @@
+const Task = require('../models/Task')
+const asyncWrapper = require('../middleware/async')
+const { createCustomError } = require('../errors/custom-error')
+const getAllTasks = asyncWrapper(async (req, res) => {
+  const tasks = await Task.find({})
+  res.status(200).json({ tasks })
+})
```

Comment:
Challenge question: what bad things might happen here if we had 1,000,000 tasks, and how could we solve that problem?